receipt/pdf_receipt.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * This example showcases the usage of creator templates to create a receipt document
+ *
+ * Run as: go run pdf_receipt.go
+ */
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"os"
+
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/creator"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io.
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
+}
+
+// Field represents field item.
+type Field struct {
+	FieldName  string `json:"FieldName"`
+	FieldValue string `json:"FieldValue"`
+}
+
+// Receipt represent Receipt object.
+type Receipt struct {
+	Title  string
+	Fields []Field
+}
+
+func main() {
```

Comment:
Probably better to use similar approach for the main like on the other examples?
like on this one
https://github.com/unidoc/unipdf-examples/blob/f46b968a6262a6509c61a8e6b723a165ed634e50/templates/trade-confirmation/pdf_trade_confirmation.go#L35 ?

Reply:
Like 
```
func main() {
  c := creator.New()
  c.SetPageMargins(50, 50, 25, 25)
  ...
  ...
  if err := c.DrawTemplate(mainTpl, data, tplOpts); err != nil {
	log.Fatal(err)
  }

  // Write output file.
  if err := c.WriteToFile("unipdf-trade-confirmation.pdf"); err != nil {
     log.Fatal(err)
  }
}

Reply:
I got it. I will update it to confirm to the earlier codes.
I have one question though.
Should I move it to the `templates` directory or should it be separate?

Reply:
Yes, I think should be moved into templates directory as this is part of it.